pkg/feature/api/feature.go

```diff
@@ -1215,6 +1275,27 @@ func (s *FeatureService) UpdateFeatureVariations(
 		}
 		return nil, dt.Err()
 	}
+	runningProgressiveRolloutExists, err := s.existsRunningProgressiveRollout(ctx, req.Id, req.EnvironmentNamespace)
```

Comment:
question:

We need to validate the existence of running progressive rollout on other function like ArchiveFeature(), CloneFeature() and so on?

Reply:
Talked at discord.